ubuntu1804/.bazelrc

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google Inc. All rights reserved..
+# Copyright 2017 Google Inc. All rights reserved.
```

Comment:
should we update the year to 2019 since we're here?

Reply:
this year needs to be when the repo was open sourced, not when this file was last updated.